tensorflow/c/c_api_experimental.h

```diff
@@ -144,7 +144,7 @@ TF_CAPI_EXPORT extern void TF_EnqueueNamedTensor(TF_Session* session,
                                                  TF_Tensor* tensor,
                                                  TF_Status* status);
 // Create a serialized tensorflow.ServerDef proto.
-TF_Buffer* TFE_GetServerDef(const char* text_proto, TF_Status* status);
+TF_CAPI_EXPORT extern TF_Buffer* TFE_GetServerDef(const char* text_proto, TF_Status* status);
```

Comment:
A check in the I/O might work well in this code. As, a system I/O failure is causing the test case failure

Reply:
Thanks for reviewing.  I don't understand what I need to change.  Do you have an example?